draft-ietf-quic-transport.md

```diff
@@ -1671,6 +1672,13 @@ disable_migration (0x0009):
   address other than that used to perform the handshake.  This parameter is a
   zero-length value.
 
+max_bytes_before_ack (0x000c):
+
+: An unsigned 32-bit integer that indicates the maximum number of bytes received
+  before an ACK frame should be sent. The peer sending this value indicates
+  to the receiver the value it should use.  If this value is absent, the
+  receiver may use any conformant acknowledgement algorithm.
+
```

Comment:
The current algorithms are either time based or packet based, requiring for example several ACKs per RTT, and specifying a maximum delay or a maximum number of packets before an ACK will be sent. What is the rationale for adding a byte based control? Also, given that the algorithms are time based or packet based, why not specify max_packets_before_ack, or max_ack_delay?

Reply:
Congestion control is byte-based, not packet based, hence the choice.  But I can see the packet vs byte based going either way.

In order to use max_ack_delay effectively, I think it's almost necessary to be able to update it during the connection.  Fraction of an RTT may not need to be updated during a connection, admittedly.  max_ack_delay also feeds into TLP and RTO timers, so making it fraction of an RTT could be a bit dangerous if one side has a poor estimate of RTT.